SECURITY.md

```diff
@@ -0,0 +1,27 @@
+# Security Policy
+
+## NOTE
+
+Whether you use this page, or add this to the bottom of your README, or add to your docs, I'd like to know how to report a security vulnerability without it being done publicly (I hope). 
+
+FYI: The rest of this doc is the default provided by github.
```

Comment:
**Reviewer Note**: This isn't a real section, but is just documenting my question, which is how do I report a security vulnerability.